Sources/KeyboardShortcuts/CarbonKeyboardShortcuts.swift

```diff
@@ -37,34 +37,32 @@ enum CarbonKeyboardShortcuts {
 	private static var hotKeyId = 0
 	private static var eventHandler: EventHandlerRef?
 
-	private static func setUpEventHandlerIfNeeded() {
-		guard
-			eventHandler == nil,
-			let dispatcher = GetEventDispatcherTarget()
-		else {
-			return
-		}
+	private static func setUpEventHandlerIfNeeded() -> Bool {
+		guard eventHandler == nil else { return true }
+		guard let dispatcher = GetEventDispatcherTarget() else { return false }
 
 		let eventSpecs = [
 			EventTypeSpec(eventClass: OSType(kEventClassKeyboard), eventKind: UInt32(kEventHotKeyPressed)),
 			EventTypeSpec(eventClass: OSType(kEventClassKeyboard), eventKind: UInt32(kEventHotKeyReleased))
 		]
 
-		InstallEventHandler(
+		guard InstallEventHandler(
 			dispatcher,
 			carbonKeyboardShortcutsEventHandler,
 			eventSpecs.count,
 			eventSpecs,
 			nil,
 			&eventHandler
-		)
+		) == noErr else { return false }
```

Comment:
I think this should throw an error instead. More Swifty. Same with `GetEventDispatcherTarget`. Then this function doesn't need to return a bool.